tests/integration-tests/clusters_factory.py

```diff
@@ -48,7 +48,7 @@ def update(self, reset_desired=False, extra_params=None):
         if extra_params:
             command.extend(["--extra-parameters", extra_params])
         command.append(self.name)
-        result = run_command(command)
+        result = run_command(command, input="y\n")
```

Comment:
what does ` input="y\n"` mean?

Reply:
It sends a "Yes" answer to pcluster update's prompt question

Reply:
It's not very intelligible imo. By looking at the param value it seems it sends a "y/n" answer to the question.
Why not having "Yes" or "y" as input value?

Reply:
It is a "y" plus a new line, like if the user entered "y" + enter. BTW now that it's implemented we could launch the command with "--yes" and get rid of the prompt.

Reply:
I just realized that the \n was a new line, it seemed the No option, like: Yes\No --> y\n.

Reply:
I'm not sure. It can be implicit for our commands but not in general, and that function is meant for running all kind of commands